tools/deb/envoy_bootstrap_v2.json

```diff
@@ -40,7 +40,30 @@
         "tag_name": "listener_address",
         "regex": "^listener\\.(((?:[_.[:digit:]]*|[_\\[\\]aAbBcCdDeEfF[:digit:]]*))\\.)"
       }
-    ]
+    ],
+    "stats_matcher": {
```

Comment:
Isn't this inclusion list too restrictive? Won't it remove stats currently scrapped by Prometheus?

Also, the health-check probe that @nmittler is working on is going to use some of the stats, so we need to retain those as well, see: https://docs.google.com/document/d/1jC0a2ku8-e3a7uMIc39HbMFbD4eh8JqWh-NFuLZSOwk/edit

Reply:
It will definitely reduce the scope of the metrics generated. However, the current Prom config we are using drops a large number of those metrics. This will save us from generating, transmitting, and then dropping those metrics now.

This covers the current set of health-checking, as best I can tell. But, if in sufficient, we can update this list from the baseline, moving forward.

Reply:
Discussed this offline, apparently Prometheus is using stats from Mixer and it doesn't scrap `/stats`.